haiku-apps/xrs/xrs-1.9.0.recipe

```diff
@@ -1,14 +1,21 @@
 SUMMARY="A simple rhythm station for creating music loops"
 DESCRIPTION="XRS is a simple Rhythm Station. You can use it to create music compositions and loops of your preferred elettronic music genres."
 HOMEPAGE="https://github.com/Freaxed/xrs-haiku"
-COPYRIGHT="2006-2022, Andrea Anzani."
+COPYRIGHT="2006-2022, Andrea Anzani.
+           2017-2020, Andrew Lindesay.
+           2012, Gerasim Troeglazov (3dEyes**).
+           2001-2010 Stephan Aßmus.
+           2006 Javier Serrano Polo.
+           1999 Fabien Fulhaber.
+           1998-1999 Be Incorporated.
+"
 LICENSE="MIT"
 REVISION="1"
 SOURCE_URI="https://github.com/Freaxed/xrs-haiku/archive/refs/tags/v1.9.0.tar.gz"
 CHECKSUM_SHA256="6ab0b77d32d3b953973ff8457ca98f2ae66e93f4951f76b6615fb9d6476992cb"
 SOURCE_DIR="xrs-haiku-1.9.0"
 
-ARCHITECTURES="x86_64 x86 !x86_gcc2"
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
If you can maybe add secondary architecture support (even if it doesn't build yet there):
```diff
ARCHITECTURES="all !x86_gcc2"
+ SECONDARY_ARCHITECTURES="?x86"
```
? marks this as untested and won't be build for secondary architecture on 32bit
If you choose to do, some $secondaryArchSuffix additions are needed, see https://github.com/haikuports/haikuports/pull/6796/files for an example on where they are needed.

Reply:
What about if I create a new 1.9.1 package with x86 fixes and rename this recipe to xrs-1.9.1 with a working secondary_arch?

Reply:
I'll clone your repo and do a local check here, will get in a bit :)